1.2-Introduction-to-Musig.ipynb

```diff
@@ -84,29 +85,34 @@
    "outputs": [],
    "source": [
     "# Compute key pairs\n",
-    "privkey0 = ECKey().generate()\n",
-    "privkey1 = ECKey().generate()\n",
-    "privkey2 = ECKey().generate()\n",
-    "pubkey0 = privkey0.get_pubkey()\n",
-    "pubkey1 = privkey1.get_pubkey()\n",
-    "pubkey2 = privkey2.get_pubkey()\n",
-    "pubkeys = [pubkey0, pubkey1, pubkey2]\n",
+    "privkey1, pubkey1 = generate_key_pair(hashlib.sha256(b'key0').digest())\n",
```

Comment:
I don't think it was documented that `generate_key_pair` had an optional argument

Reply:
Add a comment